src/state/slices/portfolioSlice/utils.ts

```diff
@@ -419,3 +423,8 @@ export const isAssetSupportedByWallet = (assetId: AssetId, wallet: HDWallet): bo
       return false
   }
 }
+
+export const getShapeshiftValidatorFromAccountSpecifier = (accountSpecifier: AccountSpecifier) => {
+  if (accountSpecifier.includes('osmosis')) return SHAPESHIFT_OSMOSIS_VALIDATOR_ADDRESS
```

Comment:
this is really brittle and non-exhaustive - could i ask this is changed to a `Record<ChainId, string>` and the function returns `string` including empty string?

Reply:
It seems weird to return a record when we only need 1 value. Can you show what you mean?